src/styles/blocks.scss/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
```

Comment:
The border-radius property is not necessary for this task. According to the task description and the checklist, you don't need to set the border-radius for the stars.